Packs/XQLDSHelper/Scripts/XQLDSHelper/XQLDSHelper.py

```diff
@@ -1768,7 +1788,7 @@ def __child_paths(
             for path in paths:
                 pit = iter(path)
                 nit = (next(pit, '\\') if c == '\\' else '' if c == '.' else c for c in pit)
-                if parent == ''.join(iter(lambda: next(nit), '')):
+                if parent == ''.join(itertools.takewhile(lambda x: x != "", iter(nit))):
```

Comment:
Why did you change it?

Reply:
@MosheEichler ruff reported an error on the line during the pre-commit, as shown below.  The change is meant to avoid the error while keeping the same logic.

```
ruff-py3.12..............................................................Failed
Packs/XQLDSHelper/Scripts/XQLDSHelper/XQLDSHelper.py:1791:56: B023 Function definition does not bind loop variable `nit`
```